### ISSUE TITLE
files generated with `write_ods` give errors when opened with Excel

### ISSUE THREAD

**Issue by pbrohan** [ropensci/readODS#120]
The files still open, but give errors on opening. 

Re-saving in Excel (I presume with the errors corrected) 
1. removes the `office:value` attribute from cells containing strings, which is indeed not in the spec
2. adds a `table:style-name` attribute to the rows
3. adds blank rows and columns to pad the sheet to the max size
4. updates the ODS spec version number to 1.3 (seems unlikely to be the cause)

I'll investigate when I have a little more time.

**Comment by pbrohan** [comment]
In general it might also be worth using the spare time to check that written documents conform to [https://docs.oasis-open.org/office/OpenDocument/v1.3/os/part3-schema/OpenDocument-v1.3-os-part3-schema.html#__RefHeading__1417678_253892949](the 1.3 spec), but this is less important, so we'll see how things go!

**Comment by chainsawriot** [comment]
@pbrohan I only have access to MS Excel on my iPhone and indeed the file can't be opened (whilst another file generated by LibreOffice can).

**Comment by pbrohan** [comment]
No problem.

I have a couple of versions of excel, so I’ll try and make the changes
tomorrow until it works!

**Comment by andreamazzella** [comment]
Having a similar issue.

Reprex:

R version 4.3.0
readODS version 1.8.0
MS Windows

``` r
library(readODS)
data.frame(id= c("3", "4"),
           value = c("C", "D")) |> 
    write_ods("test3.ods")
```

This:

- correctly produces test3.ods
- however, when I open it with MS Excel, I see this:

id | 3
-- | --
3 | 4
4 | D

Instead of the expected

id | value
-- | --
3 | C
4 | D

In my more complex case, it has also replaced some values in a column with values taken from another column, seemingly at random (not across all rows).

**Comment by pbrohan** [comment]
After much testing, the problem appears to be something to do with `zip`. Making a new zip file, moving all of the files out of the written ods file to the new zip file, then saving this as .ods seems to create a file that opens without errors. 

I've reverted to using `utils::zip` in #121 which seems to work.

@andreamazzella your issue is a separate one but should also be fixed by the upcoming v 2.0. Your reprex works correctly on my machine using the dev version!

**Comment by pbrohan** [comment]
I think this is fixed by #121 and #124, so I'll close this.
